README.md

```diff
@@ -222,6 +223,45 @@ EXAMPLE
 
 Refer to Code: *[src/commands/aem-migration/index-converter.js](./src/commands/aem-migration/index-converter.js)*
 
+## Command: `aio aem-migration:workflow-migrator`
+
+This command migrates asset processing workflows from on-premise or AMS deployments of AEM to processing profiles and OSGi configurations for use in AEM Assets as a Cloud Service..
+
+Please see [here](https://github.com/adobe/aem-cloud-migration/blob/master/README.md)
+ to know more about how the tool works, known limitations and what needs to be handled manually.
+
+### Configurations
+
+Refer to [config](./config/README.md)
+for more info.
+
+```
+USAGE
+  $ aio aem-migration:workflow-migrator
+
+OPTIONS
+  --help                     show help
+
+DESCRIPTION
```

Comment:
This description is what is specified in the `WorkflowMigratorCommand.description` in the `workflow-migrator.js`, and is shown when you execute `aio aem-migration:workflow-migrator --help`. 
So we should add the details in `WorkflowMigratorCommand.description` also.